torchx/lib/torchx/backend.ex

```diff
@@ -51,7 +51,7 @@ defmodule Torchx.Backend do
   def iota(out, axis \\ nil)
 
   def iota(%T{shape: {}, type: type} = out, nil) do
-    NIF.scalar_tensor(0, torch_type(type)) |> from_ref(out)
+    NIF.scalar_tensor(0.0, torch_type(type)) |> from_ref(out)
```

Comment:
Does this work for both `iota(s64)` and `iota(f64)`?

Reply:
Yes.